### ISSUE TITLE
Cannot download settings

### ISSUE THREAD

**Issue by theoxuan** [shanalikhan/code-settings-sync#110]
**Visual Studio Code Version :** 1.5.2
**Code Sync Settings Version :**2.2.7 
**Operating System :** OSX 10.11.6/Windows 10

![image](https://cloud.githubusercontent.com/assets/5442563/18755158/a2dc75de-811d-11e6-9d8e-4170338bec40.png)

I got this error after updating Code Sync weeks ago. And it happened to all my four computers(two of them running windows and the other OSX). I waited for a new release but still got this error each time launching VSCode. So I uninstalled Code Sync to avoid the annoying popups. Today I reinstalled it on one of my windows PC and did a upload however no errors popped up so that I thought it was fixed somehow and later I reinstalled in on one of mac. It still happens even after reinstalling and resetting config. I really don't know what is wrong.

```
//Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:121 Cannot read property 'content' of undefined: TypeError: Cannot read property 'content' of undefined
    at /Users/sean/.vscode/extensions/Shan.code-settings-sync-2.2.7/out/src/extension.js:427:101
    at next (native)
    at fulfilled (/Users/sean/.vscode/extensions/Shan.code-settings-sync-2.2.7/out/src/extension.js:4:58)
    at process._tickCallback (internal/process/next_tick.js:103:7)e.onUnexpectedError @ //Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:121
```

**Comment by shanalikhan** [comment]
can you test now on 2.3.0 version ?

**Comment by theoxuan** [comment]
![image](https://cloud.githubusercontent.com/assets/5442563/18805545/770695da-8243-11e6-867b-df1f42154515.png)

Still the same :(

**Comment by shanalikhan** [comment]
Now there is something more information
> Data is Empty for Null.

Thanks, will look more into it.

**Comment by theoxuan** [comment]
I cleared all personal settings and cache files of VSCode and then made a re-install and a re-upload and a re-download, everything's fine now.

- Guess the glitch is caused by some ~~dead plugins~~. 

  Before cleaning I checked my **.vscode/extensions** folder and found a couple of plugins which have been uninstalled previously but somehow still lay there.
- Also I switched to a **new gist** just in case.

Hope these help.
